lib/bloc/sentiment/sentiment_state.dart

```diff
@@ -13,13 +13,16 @@ class LoadedSentimentState extends SentimentState {
   LoadedSentimentState({
     required this.positiveSentiment,
     required this.negativeSentiment,
+    required this.sentimentOverview,
```

Comment:
Revert all changes in this file and move the changes into a new state file `sentiment_overview_state.dart` which will be the states for `SentimentOverviewBloc`